Makefile

```
@@ -49,9 +49,10 @@ go-version-check:
 		( $${GO_VERSION_ARR[1]} -lt $${GO_VERSION_REQ[1]} ||\
 		( $${GO_VERSION_ARR[1]} -eq $${GO_VERSION_REQ[1]} && $${GO_VERSION_ARR[2]} -lt $${GO_VERSION_REQ[2]} )))\
 	]]; then\
-		echo "SDA requires go $${GO_VERSION_MIN} to build; found $${GO_VERSION}.";\
-		exit 1;\
-	fi;
+		echo "SDA requires go $${GO_VERSION_MIN} to build; found $${GO_VERSION}."; \
+		exit 1; \
+	fi; \
+	echo "GO version: $${GO_VERSION}.";
```

```suggestion
	echo "GO version: $${GO_VERSION}."
```

The last line shouldn't need a semicolon